### ISSUE TITLE
drag action on mobile does not work with hammerjs library

### ISSUE THREAD

**Issue by pterofractal** [DevExpress/testcafe#5000]
### What is your Test Scenario?
My team owns a product that uses hammerjs to capture customer interactions. Namely, if a customer tries to swipe to a new page. I am trying to write a test for mobile browsers that can test this behavior but on mobile the `drag` action does nothing
 
### What is the Current behavior?
Doing a `drag` action on an element through a mobile browser does nothing. I _think_ it is just trying to scroll the page
 
### What is the Expected behavior?
The object I am trying to drag should be dragged.
 
### What is your web application and your TestCafe test code?
I can't share my actual application code but I created a test against the [hammerjs homepage](https://hammerjs.github.io/) 

 
```js
fixture`Test HammerJS`.page("https://hammerjs.github.io/");
test.only('Does Drag work', async (t) => {
  const addTouchEvents = ClientFunction(() => {
    const canvas = document.getElementById('hitarea');
    ['touchmove', 'touchstart', 'touchend', 'mousemove', 'mousedown', 'mouseup'].forEach((event) => {
      canvas!.addEventListener(event, (e) => {
        console.log(event);
        console.log(e);
      });
    });
  });
  await addTouchEvents();
  const hitArea = Selector('#hitarea');
  await t
    .debug()
    .click(hitArea) // This works on mobile
    .debug()
    .doubleClick(hitArea) // This works on mobile
    .debug()
    .drag(hitArea, 0, 500, { speed: 0.5 }) // This does not work on mobile
    .debug();
}); 
```
</details>

<details>
<summary>Your complete configuration file (if any):</summary>

```
{
  "pageLoadTimeout": 1000,
  "selectorTimeout": 1000,
  "assertionTimeout": 1000,
  "reporter": [
    {
      "name": "spec"
    }
  ]
}
```
</details>

<details>
<summary>Your complete test report:</summary>

```
// N/A. The test I wrote will always pass but if you watch it you will see that it isn't actually dragging the object
```
</details>

### Steps to Reproduce:

1. Visit https://hammerjs.github.io/ using a mobile device
2. Try to drag the white box 
3. Notice that Testcafe can't actually drag that white box
 
### Your Environment details:
 
* testcafe version: 1.8.3                   
* node.js version: 13.6.0                    
* command-line arguments: testcafe chrome  
* browser name and version: Chrome 80.0.3987.163 
* platform and version: macOS 10.14.6         
* other: The test above will _not_ fail. It is just meant to showcase that certain interactions on the white box work but dragging does not.

**Comment by SergeyShurygin** [comment]
I've reproduced this behavior using Safari on iOS. We will look into this.

**Comment by joshhansen** [comment]
Not specifically with hammerjs, but on a component of my own I found the testcafe drag action was failing to produce the desired result on mobile.

It turned out that the drag action was producing touch events only (touchstart, touchmove, etc.) while my component was reacting to pointer events (pointerdown, pointermove, etc.)

I believe this is an incorrect behavior by testcafe and filed #6810

To kind of work around, I added touch event handling to my widget, but this is unsatisfactory as it requires additional code that targeting the pointer events api is meant to avoid. In my view testcafe needs to better emulate touch devices by producing pointer events in addition to touch events.
